README.md

```diff
@@ -1,10 +1,20 @@
-[Quartz.NET](http://quartznet.sourceforge.net/) is one of the most powerful scheduling libraries for .NET. A contrib project has been created to integrate Quartz.NET with Castle Windsor.
+# Castle Facility for Quartz.NET
 
-* [Source code]( https://github.com/castleprojectcontrib/QuartzNetIntegration)
-* [Binaries](https://github.com/castleprojectcontrib/QuartzNetIntegration/downloads)
-* [Quartz.Windsor on NuGet](http://www.nuget.org/packages/Quartz.Windsor/)
-* [Documentation](https://github.com/castleprojectcontrib/QuartzNetIntegration/tree/master/docs)
+## Introduction
+The Castle Facility for Quartz.NET provides a facility for the Quartz.NET job scheduler that easily integrates into Castle Windsor.
 
-More information about this facility:
+## Features
+* Provides a facade around Quartz.NET to schedule and manage jobs.
+* Facility can configure Quartz.NET easily through XML-configuration file.
 
-* [Windsor facility for Quartz.NET by Mauricio Scheffer](http://bugsquash.blogspot.com/2009/03/windsor-facility-for-quartznet.html)
\ No newline at end of file
+## Dependencies
+* Castle.Core 4.2.1
+* Castle.Windsor 4.1.0
+* Quartz 3.0.2
+
+## NuGet Package
+* https://www.nuget.org/packages/Quartz.Windsor/
```

Comment:
I assume once we get the next release out we'll change this to Castle.Facilities.Quartz, I didn't get a reply from Mauricio about ownership of the old package. The old name is bad anyway because it sounds like the Quartz project maintains the package because of the order.

Reply:
At this moment, it is out of my power to change the name, but hopefully we can release a newer version of this package as soon as possible

Reply:
You actually did change the package name 😉, but I think that is good, we'll release with a new name.

```
<PackageId>Castle.Facilities.Quartz</PackageId>
```